src/main/java/com/hbm/inventory/fluid/Fluids.java

```diff
@@ -182,7 +185,8 @@ public class Fluids {
 	public static final FT_NoID NOID = new FT_NoID();
 	public static final FT_Delicious DELICIOUS = new FT_Delicious();
 	public static final FT_Leaded LEADED = new FT_Leaded();
-	
+
+	public static final FT_NoDispersable NO_DISPERSE= new FT_NoDispersable();
```

Comment:
Reminder, that's what the VISCOUS trait is for

Reply:
On main, `FT_Viscous` isn't in use anywhere as it was only meant to mark fluids that can't be turned into a mist or cloud, a feature which doesn't exist yet. As for antimatter, simply add a second exception for the `FT_Amat` trait.